torchtext/data/metrics.py

```diff
@@ -79,11 +80,11 @@ def bleu_score(candidate_corpus, references_corpus, max_n=4, weights=[0.25] * 4)
 
         clipped_counter = candidate_counter & reference_counters
 
-        for ngram in clipped_counter:
-            clipped_counts[len(ngram) - 1] += clipped_counter[ngram]
+        for ngram, count in clipped_counter.items():
+            clipped_counts[len(ngram) - 1] += count
 
-        for ngram in candidate_counter:  # TODO: no need to loop through the whole counter
-            total_counts[len(ngram) - 1] += candidate_counter[ngram]
+        for i in range(max_n):
```

Comment:
Can you add a one-line comment explaining the change?

Reply:
Thank you for your review! Sure!